extensions/positron-r/src/session.ts

```diff
@@ -670,8 +670,8 @@ export class RSession implements positron.LanguageRuntimeSession, vscode.Disposa
 	public async activateLsp() {
 		// Start LSP for the foreground session only if its been previously stopped
 		if (this._lsp.state === LspState.stopped) {
-			this._queue.add(async () => {
-				this.startLsp();
+			await this._queue.add(async () => {
+				await this.startLsp();
```

Comment:
In hindsight, I don't think we need `activateLsp` to resolve once the LS is activated. We only need that for `deactivateLsp`.